python/tvm/script/context_maintainer.py

```diff
@@ -16,59 +16,166 @@
 # under the License.
 """TVM Script Context Maintainer for TIR"""
 
-from tvm.te import schedule
+from typing import List, Mapping, Union, Optional, Dict, Callable
+import synr
+
+
+import tvm
+from tvm.ir import Span
+from tvm.tir import Var, Buffer, PrimExpr, Stmt, MatchBufferRegion
+from tvm.runtime import Object
+from .node import BufferSlice
+
+
+class BlockInfo:
+    """Information for block and block_realize signature"""
+
+    alloc_buffers: List[Buffer] = []
+    """List[Buffer]: alloc_buffers list for the block"""
+    match_buffers: List[MatchBufferRegion] = []
+    """List[MatchBufferRegion]: match_buffer_region list for the block"""
```

Comment:
Please document this more instead of just restating what the variable is called. You've done the same thing on the other variables.

Maybe it would be helpful to provide an example of a block and show what each variable corresponds to.